tests/storage/merge_test.py

```diff
@@ -539,10 +539,13 @@ def test_reduce_chunked_leaf(self):
             fake_sd = env.sdcache.domains[env.sd_manifest.sdUUID]
             fake_base_vol = fake_sd.produceVolume(subchain.img_id,
                                                   subchain.base_id)
-
-            assert fake_base_vol.__calls__ == [
-                ('reduce', (base_vol.optimal_size(as_leaf=True),), {}),
-            ]
+            # Chunked leaf volumes add one chunk of free space as condition
+            # to reduce, so the VM will not pause quickly when it is started.
+            # Consequently, cold merges will not call reduce.
+            calls = getattr(fake_base_vol, "__calls__", {})
+            # Verify that 'calls' is empty which means that 'reduce' wasn't
+            # called
+            assert len(calls) == 0
```

Comment:
A simpler way would be:

    assert fake_base_vol.__calls__ == []
    
And if the list is set only on the first call:

    assert getattr(fake_base_vol, “__calls__”, []) == []

Reply:
The list is only set on first call, yes.

I opted for this solution for consistency with the check in the next test. Also, `__calls__` (if set) would be a dict, so the current solution does not re-type the member. But I'm ok with your proposed change.